### PR TITLE
feat: light mode only alt css

### PR DESCRIPTION
**Pull request by danny007in**
Fixes https://github.com/ColorlibHQ/AdminLTE/issues/3530

### CODE REVIEW

File: build/scss/mixins/_accent.scss [hunk 1]
```diff
@@ -91,12 +91,14 @@
...
     }
   }
-  .dark-mode.accent-#{$name} {
-    .page-item {
-      .page-link {
-        &:hover,
-        &:focus {
-          color: lighten($link-color, 5%);
+  @include dark-mode () {
+    .accent-#{$name} {
```
Comment (REJack) [inline build/scss/mixins/_accent.scss:95]: this should be `&.accent-#{$name} {` or not?

**Review by REJack — CHANGES_REQUESTED** [review]

File: build/scss/mixins/_accent.scss (hunk 1 above)
Comment (danny007in) [reply build/scss/mixins/_accent.scss:95]: result will be same 😃

Comment (danny007in) [reply build/scss/mixins/_accent.scss:95]: once will check

Comment (danny007in) [reply build/scss/mixins/_accent.scss]: FIxed

**Comment by danny007in** [comment]
jack 
edit this pr like bundlewatch
then merge

**Review by REJack — APPROVED** [review]

**Pull request merged**